### ISSUE TITLE
Unexpected report when using ClassName=UpperCamel

### ISSUE THREAD

**Issue by keineahnung2345** [dougpuob/cppnamelint#71]
I am using release v0.3.4 on Windows 10, following is my `cppnamelint.toml`:

```
...
ClassName               = 1 # 0: Default (UpperCamel)
                            # 1: UpperCamel
...
```
Either I set this value to 0 or 1, it will wrongly report that my class name violates the rule, following is the formatted `result.json`:
```
            {
                "Column": 1,
                "Expected": "",
                "Line": 46,
                "TargetName": "XxxxXxxx",
                "Type": 5,
                "TypeName": "class"
            }
```

**Comment by dougpuob** [comment]
@keineahnung2345 glad to receive your feedback. I am planning to archive this project because the major feature of this project had been merged into clang-tidy (https://github.com/llvm/llvm-project/commit/2b9b5bc0).

**Comment by keineahnung2345** [comment]
@dougpuob Oh, it's a good news.
